snippets/osdk-deprecation.adoc

```diff
@@ -43,7 +43,7 @@
 [IMPORTANT]
 ====
 [subs="attributes+"]
-The Operator SDK CLI tool, including the related scaffolding and testing tools for Operator projects, is deprecated with the {product-title} {product-version} release. Operator SDK is still released with {product-title} {product-version} and continues to be supported; however, it will stop receiving updates and no longer be released with a future version of this product. It is not recommended for new deployments.
+The Red Hat-supported version of the Operator SDK CLI tool, including the related scaffolding and testing tools for Operator projects, is deprecated with the {product-title} {product-version} release. The Red Hat-supported version of the Operator SDK is still released with {product-title} {product-version} and continues to be supported; however, it will stop receiving updates and no longer be released with a future version of this product. The Red Hat-supported version of the Operator SDK is not recommended for new deployments.
```

Comment:
IDK that 'deprecated' is the right word to use in this context. Deprecated in this context means "to discouragement of use of something". I think what is actually happening here is that Red Hat is no longer providing/supporting a downstream distribution of the Operator SDK cli.

Reply:
If I understand correctly, the plan is to stop shipping a downstream release of the Operator SDK in 4.17. So, I think that _deprecation_ would apply for 4.16. https://issues.redhat.com/browse/OCPSTRAT-1440